CONTRIBUTING.adoc

```diff
@@ -0,0 +1,184 @@
+
+
+=== Fixing Issues (Pull Requests)
+
+////
+This could use a pass to be more focused on what a PR submitter should do at the start of the process.
+////
+
+Here’s what generally happens when you open a PR against the `knative/docs` repo:
+
+. One of the assigned repo maintainers will triage the PR by assigning relative priority, adding appropriate labels, and performing an initial documentation review.
+. If the PR involves significant technical changes, for example new features, or new and changed sample code, the PR is assigned to a Subject Matter Expert (SME), typically an engineer working on Knative, for technical review and approval.
+. When both the technical writers and SMEs are satisfied with the quality of the writing and the technical accuracy of the content, the PR can be merged. A PR requires two labels before it can merge: `lgtm` and `approved`.
+* The SME is responsible for reviewing the technical accuracy and adding the `lgtm` label.
+* The https://github.com/knative/docs/blob/master/OWNERS_ALIASES[Knative technical writers] are who provide the `approved` label when the content meets quality, clarity, and organization standards (see link:#style-guide[Style Guide]).
+
+We appreciate contributions to the docs, so if you open a PR we will help you get it merged. You can also post in the `#docs` https://knative.slack.com/[Slack channel] to get input on your ideas or find areas to contribute before creating a PR.
+
+== Getting involved
+
+There are a couple of different ways to get started with contributing to the Knative documentation:
+
+* Look for a https://github.com/knative/docs/labels/kind%2Fgood-first-issue[Good First Issue] in the backlog.
+
+* Try out Knative and send us feedback. For example, run through the link:./docs/install/[install guides] and then try link:./docs/serving/getting-started-knative-app.md[Getting Started with Knative Serving] or link:./docs/eventing/getting-started.md[Getting Started With Eventing].
++
+Keep a https://devrel.net/developer-experience/an-introduction-to-friction-logging[friction log] of your experience and then use that to open your first set of PRs. Examples:
+** What was difficult for you?
+** Did you stumble on something because the steps weren’t clear?
+** Was a dependency not mentioned?
+
+*A few pointers and other considerations*
+
+* Think about how you can improve the documentation as a whole. For example, how can you fix the issue you found so that others don’t run into the same challenges?
+* Are there multiple places the issue could be fixed?
+** Are there other pages that you should apply your update?
+** Would it help if there was a link to more or related information on a page? On a related page?
+* Was the title or description misleading? Did you expect to find something different in that docs section?
```

Comment:
I think you could probably remove this for the sake of brevity? Not that it isn't helpful, it's just already a long doc and this isn't the most valuable bit

Reply:
SGTM; this was copy-pasta from the old doc, like the other two areas that I commented needed a pass.

Reply:
sick